src/sentry/api/endpoints/group_notes_details.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import
+
+from rest_framework import status
+from rest_framework.exceptions import PermissionDenied
+from rest_framework.response import Response
+
+from sentry.api.bases.group import GroupEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers import serialize
+from sentry.api.serializers.rest_framework.group_notes import NoteSerializer
+from sentry.models import Activity
+
+
+class GroupNotesDetailsEndpoint(GroupEndpoint):
+    # We explicitly don't allow a request with an ApiKey
```

Comment:
/cc @mitsuhiko @dcramer 

I wasn't entirely sure how to handle this behavior since I don't think we should allow an `ApiKey` to edit/delete any comments.

If we think that's ok, I can just make this do the right thing.

But in the meantime, I'm just going to err on being cautious and block it from being allowed.